circle.yml

```diff
@@ -4,12 +4,14 @@ machine:
   xcode:
     version: "7.3"
 dependencies:
-  pre:
-    - brew install shellcheck
+  # pre:
+    # - brew install shellcheck
   override:
     - bundle install --jobs=4 --retry=3
 test:
   pre:
     - ulimit -n 400
   override:
+    - bundle exec danger # for quick, instant feedback of PR body and title
```

Comment:
depending on what you want to do here, you could consider using `-- danger_id="comment" and `-- danger_id="results" - it would show up as two separate messages